osclib/remote_package.py

```diff
@@ -0,0 +1,43 @@
+
+import logging
+import osc.core
+import osc.conf
+
+from lxml import etree as ET
+from urllib.error import HTTPError, URLError
+
+class PackageNotFound(Exception):
+    """ The package was not found in the build service """
+    pass
+
+class RemotePackage(object):
+    """ This class represents a package on the build service side """
+    def __init__(self, name, project=None):
+        self.name = name
+        self.project_name = project
+
+    def copy(self, target_project_name, expand=False):
+        apiurl = osc.conf.config['apiurl']
+        osc.core.copy_pac(apiurl, self.project_name, self.name, apiurl, target_project_name, self.name, expand=expand)
```

Comment:
note that copy_pac is greatly expensive operating. For such use cases a revisioned _link is much clearly preferred.

Reply:
yeah, I expect it is expensive. I just want to avoid any diff issue. Revisioned _link can be solution as I expect in SLE15 it won't change much. If we want to use it on staging it can be more tricky as staging change more frequently and link can be broken. ( question is how big issue it is as it means that it is removed from staging )

Reply:
if you plan to rebuild all of SLE15 history against a moving target you want to go back to the drawing board anyway.